include/juice/juice.h

```diff
@@ -72,6 +79,7 @@ typedef struct juice_turn_server {
 	const char *username;
 	const char *password;
 	uint16_t port;
+	juice_transport_t transport;
```

Comment:
This is an issue now as it breaks the library interface and would require releasing a new major version.